compiler/lib/compiler.go

```diff
@@ -261,7 +266,8 @@ func (c *Compiler) load(filename string) (*config, error) {
 
 func (c *Compiler) GetLoader() *starlarkLoader {
 	modules := getModules()
-	modules["remote_repo"] = starlark.NewBuiltin("remote_repo", c.ModuleService.Add)
+	modules["ConfigRollout"] = starlark.NewBuiltin("ConfigRollout", newConfigRollout)
```

Comment:
Can we have the "ConfigRollout"  "RolloutStage" ets defined as consts?

Reply:
done.